css-typed-om/Overview.bs

```diff
@@ -2903,6 +2903,49 @@ depending on the property they came from:
         return "currentcolor".
     2. Otherwise, return the result of serializing the <<color>> value.
 
+: 'stroke-align'
+::
+    1. Return the string representation of the keyword:
+        * center,
+        * inset,
+        * outset.
```

Comment:
Yeah, no need to list these explicitly. Just say "return the result of serializing the keyword value".